src/styles/todoapp.scss

```diff
@@ -131,3 +131,50 @@
     }
   }
 }
+
+.item-enter {
+  max-height: 0;
+}
+
+.item-enter-active {
```

Comment:
Class names should follow the naming convention with double underscores for nested elements and double dashes for modifiers. Consider renaming '.item-enter-active' to '.item--enter-active'.